modules/schedule/main.tf

```diff
@@ -0,0 +1,10 @@
+resource "opsgenie_schedule" "default" {
+  count = module.this.enabled ? 1 : 0
+
+  name          = var.schedule.name
+  description   = try(var.schedule.description, var.schedule.name)
```

Comment:
if `schedule.description` is null, it will be used.
Use the pattern to check for `null` you used in the PR 27